modules/nested-accordion/module.php

```diff
@@ -27,5 +26,21 @@ public function __construct() {
 				'nested-elements',
 			], ELEMENTOR_VERSION, true );
 		} );
+
+		add_action( 'elementor/frontend/after_register_styles', [ $this, 'register_style' ] );
```

Comment:
@rami-elementor - the before action is "/editor" while the after is "/frontend" is it intended?

Reply:
The previous action is for JS, this one is for CSS.

When registering styles, there are specificity issues, therefor we need to position them after the all frontend styles are loaded.

Furthermore, when it was in the "_modules.scss" file, it was loaded in the end, after all fronend atyles are loaded.

Reply:
@rami-elementor The use of this action hook is causing some specificity problems. That is why some of the nested tabs tests are failing.

I would prefer if we can enqueue the style in the `head `of the page, but after all other styles. I am not sure if we can do that?

Reply:
@rami-elementor @rotemee @louiswol94 
I have worked on a possible solution to enqueue the widgets in this order:
- frontend.css
- conditionally loaded widgets
- inline widget styling

Reply:
@hein-obox how do you suggest to load the style in the head?

Reply:
@rami-elementor Sorry, I think I had forgotten to share my test PR: 
https://github.com/elementor/elementor/pull/27944. 

My idea was to include the active widget CSS with this hook: `'elementor/frontend/enqueue_widgets'`